Tasks/AzureKeyVaultV1/operations/azure-arm-keyvault.ts

```diff
@@ -176,21 +179,41 @@ export class KeyVaultClient extends azureServiceClient.ServiceClient {
                 '{secretName}': secretName
             }
         );
-
         console.log(tl.loc("DownloadingSecretValue", secretName));
-        this.invokeRequest(httpRequest).then(async (response: webClient.WebResponse) => {
-            if (response.statusCode == 200) {
-                var result = response.body.value;
-                return new azureServiceClientBase.ApiResult(null, result);
-            }
-            else if (response.statusCode == 400) {
-                return new azureServiceClientBase.ApiResult(tl.loc('GetSecretFailedBecauseOfInvalidCharacters', secretName));
-            }
-            else {
-                return new azureServiceClientBase.ApiResult(azureServiceClientBase.ToError(response));
-            }
-        }).then((apiResult: azureServiceClientBase.ApiResult) => callback(apiResult.error, apiResult.result),
-            (error) => callback(error));
+        this.invokeRequestWithRetry(httpRequest,secretName)
+    .then((apiResult: azureServiceClientBase.ApiResult) => callback(apiResult.error, apiResult.result))
```

Comment:
Please fix indentation here and on the next ine

Reply:
Fixed the indentation